.github/workflows/ci.yml

```
@@ -230,10 +230,14 @@ jobs:
     if: ${{ github.event_name != 'pull_request' || needs.ci-path-changes.outputs.has_code_changes == 'true' }}
     steps:
       - uses: actions/checkout@de0fac2e4500dabe0009e67214ff5f5447ce83dd # v6.0.2
-        with: { fetch-depth: 1 }
+        with: { fetch-depth: 0 }
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)